microbial_thermodynamics/scipy_demo.py

```diff
@@ -50,10 +49,20 @@ def full_equation_set(
         t,
         a=y[2 * number_of_species : 3 * number_of_species],
         R=y[number_of_species : 2 * number_of_species],
+        s=y[3 * number_of_species],
+        w=y[3 * number_of_species + 1],
+        reaction_energy=reaction_energies,
+        v=np.array([1.0, 2.0]),
```

Comment:
This and the one below should be `v=np.array([1.0, 1.0])` (this is unrelated to the current problem, it's just a mistake that I noticed)